src/pfe/file-watcher/server/test/scripts/utils.sh

```diff
@@ -22,10 +22,15 @@ export TEST_INFO_DIR="$HOME/.codewindtest/turbine"
 # Function to get the che and keycloak endpoint
 function getEndpoints() {
     echo -e "${BLUE}Getting che endpoints${RESET}\n"
-
-    CHE_ENDPOINT=$(kubectl get routes --selector=component=che -o jsonpath="{.items[0].spec.host}" 2>&1)
-    KEYCLOAK_HOSTNAME=$(kubectl get routes --selector=component=keycloak -o jsonpath="{.items[0].spec.host}" 2>&1)
-    TOKEN_ENDPOINT="http://${KEYCLOAK_HOSTNAME}/auth/realms/che/protocol/openid-connect/token"
+    if [ ! -z "$DISABLE_SSL" ]; then
+        PROTOCOL="http://"
+    else
+        PROTOCOL="https://"
+    fi
```

Comment:
actually a small nit.

Can you have ENABLE_SSL instead of DISABLE_SSL, it makes more sense and is the convention usually